assets/css/app.css

```diff
@@ -36,7 +36,7 @@
 
 @media (min-width: 26em) {
   .comment-minw {
-    min-width: 91.45%;
+    min-width: 94.2%;
```

Comment:
oh the _precision_ ... 🙄 
excuse my ignorance, but could we use Tachyons' `w-90` for this? http://tachyons.io/docs/layout/widths/